megamek/src/megamek/common/MechSummaryCache.java

```diff
@@ -132,8 +152,8 @@ public void removeListener(Listener listener) {
     }
 
     private MechSummary[] m_data;
-    private Map<String, MechSummary> m_nameMap;
-    private Map<String, MechSummary> m_fileNameMap;
+    private final Map<String, MechSummary> m_nameMap;
+    private final Map<String, MechSummary> m_fileNameMap;
     private Map<String, String> hFailedFiles;
     private int cacheCount;
     private int fileCount;
```

Comment:
Pre-existing, but the location of these is odd. It should be at the top of the file.